client/src/components/StreetView.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { withGoogleMap, StreetViewPanorama } from 'react-google-maps';
+import Browser from '../util/browser';
+
+export type StreetViewAddr = {
+  bbl: string,
+  lat: number,
+  lng: number
+};
+
+export type StreetViewProps = {
+  addr: StreetViewAddr|null|undefined,
+};
+
+type ImplProps = {
+  heading: number,
+  coordinates: google.maps.LatLng|undefined
+};
+
+type State = ImplProps;
+
+const UnwrappedStreetViewImpl: React.FC<ImplProps> = props => {
+  return (
+    <StreetViewPanorama
+      visible
+      defaultPosition={props.coordinates}
+      position={props.coordinates}
+      pov={{ heading: props.heading, pitch: 15 }}
+      zoom={0.5}
+      options={{
+        disableDefaultUI: true,
+        enableCloseButton: false,
```

Comment:
`enableCloseButton` appears to be a new option we need to explicitly set to `false`, or else an annoying "back" button will be shown that attempts to take the user back to the main google map (which is useless in our case, since we're using mapbox for the main map, in a different place on the page).

Reply:
nice catch!